amr-wind/wind_energy/actuator/wing/wing_ops.H

```diff
@@ -164,8 +165,19 @@ struct ComputeForceOp<
             windvector[2] = (grid.vel[ip] - wdata.vel_tr) & blade_z;
 
             const auto vmag = vs::mag(windvector);
+
+            // Calculate the aoa based on the pitch actuation table if supplied
+            // (interpolate pitch angle at step n) or the initial pitch angle
+            amrex::Real aoa;
+            if (!wdata.pitch_timetable_file.empty()) {
+                wdata.pitch = amr_wind::interp::linear(
```

Comment:
@mbkuhn At the moment the pitch angle is interpolated to the current time. Is this consistent with how the velocities, the forces and the actuator point locations are handled (between step n and n+1)? Or should one interpolate to n + 1/2?

Reply:
It should be interpolated to n+1/2, the average between current_time and new_time. Good question!